src/element/newElement.test.ts

```diff
@@ -15,79 +16,310 @@ const assertCloneObjects = (source: any, clone: any) => {
+      roughness: 1,
+      opacity: 100,
+    });
+
+    // @ts-ignore
+    element.__proto__ = { hello: "world" };
+
+    mutateElement(element, {
+      points: [
+        [1, 2],
+        [3, 4],
+      ],
+    });
+
+    const copy = duplicateElement(null, new Map(), element);
+
+    assertCloneObjects(element, copy);
 
-  // @ts-ignore
-  element.__proto__ = { hello: "world" };
+    // @ts-ignore
+    expect(copy.__proto__).toEqual({ hello: "world" });
+    expect(copy.hasOwnProperty("hello")).toBe(false);
 
-  mutateElement(element, {
-    points: [
-      [1, 2],
-      [3, 4],
-    ],
+    expect(copy.points).not.toBe(element.points);
+    expect(copy).not.toHaveProperty("shape");
+    expect(copy.id).not.toBe(element.id);
+    expect(typeof copy.id).toBe("string");
+    expect(copy.seed).not.toBe(element.seed);
+    expect(typeof copy.seed).toBe("number");
+    expect(copy).toEqual({
+      ...element,
+      id: copy.id,
+      seed: copy.seed,
+    });
   });
 
-  const copy = duplicateElement(null, new Map(), element);
+  it("clones text element", () => {
+    const element = API.createElement({
+      type: "text",
+      x: 0,
+      y: 0,
+      strokeColor: "#000000",
+      backgroundColor: "transparent",
+      fillStyle: "hachure",
+      strokeWidth: 1,
+      strokeStyle: "solid",
+      roundness: null,
+      roughness: 1,
+      opacity: 100,
+      text: "hello",
+      fontSize: 20,
+      fontFamily: FONT_FAMILY.Virgil,
+      textAlign: "left",
+      verticalAlign: "top",
+    });
 
-  assertCloneObjects(element, copy);
+    const copy = duplicateElement(null, new Map(), element);
 
-  // @ts-ignore
-  expect(copy.__proto__).toEqual({ hello: "world" });
-  expect(copy.hasOwnProperty("hello")).toBe(false);
+    assertCloneObjects(element, copy);
 
-  expect(copy.points).not.toBe(element.points);
-  expect(copy).not.toHaveProperty("shape");
-  expect(copy.id).not.toBe(element.id);
-  expect(typeof copy.id).toBe("string");
-  expect(copy.seed).not.toBe(element.seed);
-  expect(typeof copy.seed).toBe("number");
-  expect(copy).toEqual({
-    ...element,
-    id: copy.id,
-    seed: copy.seed,
+    expect(copy).not.toHaveProperty("points");
+    expect(copy).not.toHaveProperty("shape");
+    expect(copy.id).not.toBe(element.id);
+    expect(typeof copy.id).toBe("string");
+    expect(typeof copy.seed).toBe("number");
   });
 });
 
-it("clones text element", () => {
-  const element = API.createElement({
-    type: "text",
-    x: 0,
-    y: 0,
-    strokeColor: "#000000",
-    backgroundColor: "transparent",
-    fillStyle: "hachure",
-    strokeWidth: 1,
-    strokeStyle: "solid",
-    roundness: null,
-    roughness: 1,
-    opacity: 100,
-    text: "hello",
-    fontSize: 20,
-    fontFamily: FONT_FAMILY.Virgil,
-    textAlign: "left",
-    verticalAlign: "top",
+describe("duplicating multiple elements", () => {
+  it("duplicateElements should clone bindings", () => {
```

Comment:
```suggestion
  it("should clone bindings", () => {
```
As the describe block conveys the block is about duplicating elements